libcaja-private/caja-module.c

```diff
@@ -32,9 +32,6 @@
 
 #define CAJA_TYPE_MODULE    	(caja_module_get_type ())
 #define CAJA_MODULE(obj)		(G_TYPE_CHECK_INSTANCE_CAST ((obj), CAJA_TYPE_MODULE, CajaModule))
-#define CAJA_MODULE_CLASS(klass)	(G_TYPE_CHECK_CLASS_CAST ((klass), CAJA_TYPE_MODULE, CajaModule))
-#define CAJA_IS_MODULE(obj)		(G_TYPE_INSTANCE_CHECK_TYPE ((obj), CAJA_TYPE_MODULE))
-#define CAJA_IS_MODULE_CLASS(klass)	(G_TYPE_CLASS_CHECK_CLASS_TYPE ((klass), CAJA_TYPE_MODULE))
```

Comment:
These are standard defines for a GType, but as they are not needed and private, OK.

Reply:
Should i add them back?

Reply:
Ahh, it is OK . I over read your first comment :)

Reply:
No need if we're all happy not to have them.  I just wanted to mention that those might actually be useful at some point, and are not mere letfover of days long past; but as they don't appear in a header, it's OK to remove them if we want.